### DISCUSSION TITLE
Create new audio files

### DISCUSSION THREAD

**Discussion by RustoMCSpit** [linuxmint/discussions#278] [Ideas]
So under Create New Folder and Create New Document there should be 'Create New Audio File', then it opens up a simple waveform GUI and immediately starts recording and ends on a button press (like how the screenshot tool works.

.opus files would work well

**Comment by anaximeno** [comment]
Interesting, you can try to create a new spice action for that, it will be available for new versions of Mint to download like applets, desklets, and extensions. Please look at the repository https://github.com/linuxmint/cinnamon-spices-actions to see how that could be done.

**Reply by RustoMCSpit** [reply]
no im thinking it should be an integrated default feature

**Reply by claudiux** [reply]
Why impose a feature on all users that will only be useful to some?

**Reply by RustoMCSpit** [reply]
this is one of those 'you never know when youll need it' type features (like the calculator app) where everyone will need it at one point, it's function is so basic that it should be standard.

**Reply by anaximeno** [reply]
I believe an action is still the best way to go, because everyone who wants it, can easily install it when needed, the same way you can easily install an applet and add it to the panel.

**Reply by RustoMCSpit** [reply]
i mean, id argue that a whole gui application is needed by default for voice recording. this seems like a logical core functionality?

**Reply by anaximeno** [reply]
I don't know if it is core for desktops since audio recording is not one of the most frequent activities for users on desktops (or laptops), I believe. An app called `gnome-sound-recorder` can be used for that though.

**Comment by 100savage** [comment]
This might be a little helpful, but it's not exactly what you ask for. Install gnome-sound-recorder

`sudo apt install gnome-sound-recorder`

Create a file called **recorder.nemo_action** and place it in your **.local/share/nemo/actions/** 
Note: the .local folder is hidden and located in your home directory.
Put the below code in the **recorder.nemo_action** file

```
[Nemo Action] 
Name=Create New Audio File
Comment=Opens Gnome-sound-recorder GUI
Exec=gnome-sound-recorder
Icon-Name=audio-input-microphone-symbolic
Selection=any
Extensions=desktop;
```


In a terminal restart nemo

`nemo -q`

You can now right-click on your desktop and select Create New Audio File. I haven't found a way to have it immediately start recording

**Reply by claudiux** [reply]
Using `pw-record`?

**Reply by 100savage** [reply]
as far as I understand it, pw-record doesn't have a GUI and a button to stop the recording
